client/src/app/Pages/login/login.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="container vh-100 d-flex justify-content-center align-items-center">
+  <div class="card p-4 shadow-sm" style="width: 100%; max-width: 400px">
+    <h3 class="card-title text-center mb-4">Login</h3>
+    <form [formGroup]="loginForm" (ngSubmit)="handleLogin()">
+      <div class="form-group">
+        <label for="email">Email</label>
+        <input  id="email" class="form-control" formControlName="email" placeholder="Enter your email"
+          [class.is-invalid]="loginForm.controls['email'].invalid && loginForm.controls['email'].touched"/>
+
+        <div *ngIf="loginForm.controls['email'].invalid && loginForm.controls['email'].touched" class="invalid-feedback">
```

Comment:
You should directly specify for what error this message is:
```suggestion
        <div *ngIf="loginForm.controls['email'].hasError('email')" class="invalid-feedback">
```